charts/consul/values.yaml

```diff
@@ -1859,6 +1859,76 @@ connectInject:
     # Note: This value has no effect if transparent proxy is disabled on the pod.
     defaultOverwriteProbes: true
 
+  # Configures consul-cni plugin for Consul Service mesh services
+  cni:
+    # If true, then all traffic redirection setup with use the consul-cni plugin
+    # @type: boolean
+    enabled: false
+ 
+    # If multus plugin is enabled or not. If true, the CNI installer will generate a consul CNI configuration
+    # in a location for the multus program to use. Also, the helm chart will create a custom resource for
+    # NetworkAttachmentDefintion that multis will consume. The values for cniBinDir and cniNetDir must also be 
+    # changed for multus to work.
+    # Please refer to your cloud platform for more informaton on using multus.
+    # @type: boolean
+    multus: false
+
+    # Log level for the installer and plugin. Overrides global.logLevel
+    # @type: string
+    logLevel: null 
+    
+    # Location on the kubernetes node where the CNI plugin is installed. Shoud be the absolute path and start with a '/'
+    # @type: string
+    cniBinDir: "/opt/cni/bin"
+
+    # Location on the kubernetes node of all CNI configuration. Should be the absolute path and start with a '/'
+    # @type: string
+    cniNetDir: "/etc/cni/net.d"
+ 
+    # The resource settings for CNI installer daemonset.
+    # @recurse: false
+    # @type: map
+    resources:
+      requests:
+        memory: "50Mi"
+        cpu: "50m"
+      limits:
+        memory: "50Mi"
+        cpu: "50m"
+
+    # Resource quotas for running the daemonset as system critical pods
+    resourceQuota:
+      pods: 5000
```

Comment:
do we still need this? I thought you said we only needed it if running in `kube-system` namespace

Reply:
Yes we do.